Added new remove the requirement for inbound network rules section (#5010)
hashicorp/boundary: website/content/docs/concepts/connection-workflows/multi-hop.mdx

Multi-hop sessions allow you to chain together two or more workers
across multiple networks to form reverse proxy connections between the user and the target, even in complex networks with strict outbound-only policies.

> ## Remove the requirement for inbound network rules

Reviewer: ```suggestion
## Inbound network rules
```

> With a multi-hop deployment, all connections are initiated outbound from the most downstream worker in the chain. When the connection is established between the workers, the proxied connections go over the established connection.

Reviewer: ```suggestion
With a multi-hop deployment, all outbound connections are initiated from the most downstream worker in the chain. After Boundary establishes the initial connection between the workers, it uses the established connection for any subsequent connections.
```

Author: Hey @Dan-Heath I would say in the suggestion that the tunnels are established outbound, so `all outbound connections` to me suggests that there maybe inbound connections, but it's always established from the outbound direction, if that makes sense? The rest of the suggestions are great :)

Reviewer: That does make sense, thanks @dannyjknights ! I was worried that may have changed the meaning. I edited my suggestion to use your original phrasing there and include my edit to the second sentence. If this sounds good to you, I will approve and get this merged 🎉

These persistent TCP connections result in the requirement for only outbound connectivity.

> In the scenario where there may be a firewall(s) sitting between ingress and egress workers, organizations do not need to create additional inbound rules to facilitate a Boundary multi-hop deployment. This not only helps to

Reviewer: ```suggestion
If you have one or more firewalls sitting between the ingress and egress workers, you do not need to create additional inbound networking rules to facilitate a Boundary multi-hop deployment. This not only helps to
```

> simplify existing infrastructure configuration within your environment, but also ensures that your current security posture is not weakened or compromised.

Reviewer: ```suggestion
simplify your infrastructure configuration, but also ensures that your security posture is not weakened or compromised.
```
